simtools/applications/validate_file_using_schema.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+"""
+    Summary
+    -------
+    Validate file (e.g., metadata) using a json schema file.
+
+    Command line arguments
+    ----------------------
+    file_name (str)
+      input file to be validated
```

Comment:
Not sure if this is more information. A file is a data product, so this would not work. Why does 'input file to be validated' is not sufficient, doesn't it say everything?

Reply:
input file can be everything. a histogram? we can mention it has to be either a metadata or a data product. Wouldnt that be better?

Reply:
Added that it needs to be a yaml file.